cartoframes/utils/utils.py

```diff
@@ -334,7 +335,8 @@ def is_uuid(text):
 def get_credentials(credentials=None):
     from ..auth import defaults
     _credentials = credentials or defaults.get_default_credentials()
-    check_credentials(_credentials)
+    if credentials is not None:
```

Comment:
Why is this added? If `credentials` is None but you have `default_credentials` it should call `check_credentials` too.

Reply:
There are some tests that have no credentials configured, and `check_credentials` raises an exception.

Reply:
Aha, in that case, I would try to pass a real Credential instance to those specific tests.

Reply:
The failing tests were the `Map` ones, and the `Map` class doesn't accept a `Credentials` parameter, so I've used the `set_default_credentials` function and created a new `unset_default_credentails` becuase if not, the `test_source_no_credentials` test fails because it has the default credentials in the global variables.